boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -0,0 +1,240 @@
+.. _mec15xxevb_assy6853:
+
+MEC15xxEVB ASSY6853
+###################
+
+Overview
+********
+
+The MEC15xxEVB_ASSY6853 kit is a development platform to evaluate the
```

Comment:
Where can we find published information about this kit? There are no references on the Microchip site (nor on Google).  I can't find any available references for the part number you mention below either, "MEC1501 144WFBA SOLDER DC ASSY 6860".  Where is that published and where can I order one?  On the Microchip site, the [MEC1501](https://www.microchip.com/wwwproducts/en/MEC1501) is listed as a "future product".  Is this board and kit available to the public yet?  If not, it's likely not appropriate to be adding as a supported board.